### PR TITLE
add "Open the notebook in Colab"

### PR DESCRIPTION
**Pull request by makelinux**
because repository is open-sourced

### CODE REVIEW

File: notebooks/README.md [hunk 1]
```diff
@@ -26,12 +26,11 @@ Pre-requisites:
 * [Google Colab](https://research.google.com/colaboratory/faq.html)
 * A Gmail account that you're logged into. This will allow you to use Google Colab, which in the free tier will give you access to an NVidia T4 x 15GB GPU.
 
-**NOTE: At present, you'll need to download the notebook and upload it to Google Colab. To upload a notebook go to [Google Colab](https://colab.research.google.com) and you will be prompted to upload a notebook. Once this repository is open sourced, we will make an 'Open in Colab' button.**
-
 ## Running the notebook
 
 [The notebook](./Training_a_LoRA_With_Instruct_Lab.ipynb) in this folder will walk you through:
 
+1. [Open the notebook in Colab](https://github.com/instructlab/instructlab/blob/main/notebooks/Training_a_LoRA_With_Instruct_Lab.ipynb)
```
Comment (leseb) [inline notebooks/README.md:33]:
```suggestion
1. [Open the notebook in Google Colab](https://github.com/instructlab/instructlab/blob/main/notebooks/Training_a_LoRA_With_Instruct_Lab.ipynb)
```

File: notebooks/README.md [hunk 2]
```diff
@@ -26,12 +26,11 @@ Pre-requisites:
 * [Google Colab](https://research.google.com/colaboratory/faq.html)
 * A Gmail account that you're logged into. This will allow you to use Google Colab, which in the free tier will give you access to an NVidia T4 x 15GB GPU.
 
-**NOTE: At present, you'll need to download the notebook and upload it to Google Colab. To upload a notebook go to [Google Colab](https://colab.research.google.com) and you will be prompted to upload a notebook. Once this repository is open sourced, we will make an 'Open in Colab' button.**
-
 ## Running the notebook
 
 [The notebook](./Training_a_LoRA_With_Instruct_Lab.ipynb) in this folder will walk you through:
 
+1. [Open the notebook in Google Colab](https://github.com/instructlab/instructlab/blob/main/notebooks/Training_a_LoRA_With_Instruct_Lab.ipynb)
```
Comment (leseb) [inline notebooks/README.md:33]: Pardon my ignorance but how is this going to open the file automatically in Colab? Thanks!

Comment (makelinux) [reply notebooks/README.md]: Fixed. Thank you!

Comment (leseb) [reply notebooks/README.md]: Ahhh this looks more like it now :)

**Review by leseb — APPROVED** [review]

**Review by russellb — APPROVED** [review]
thanks!

**Pull request merged**